client/js/components/assessments/item.jsx

```diff
@@ -7,211 +7,124 @@ import UniversalInput         from "./universal_input";
-      return <div><i className="glyphicon glyphicon-exclamation-sign"></i> You left question(s) {state.unAnsweredQuestions.join()} blank. Use the "Progress" drop-down menu at the top to go back and answer the question(s), then come back and submit.</div>
-    }
-
-    return "";
-  }
-
-  getConfidenceLevels(level){
-    if(level){
-      var levelMessage = <div><b>How sure are you of your answer? Click below to move forward.</b></div>;
-      return    (<div className="confidence_wrapper">
-                  {levelMessage}
-                  <input type="button" className="btn btn-check-answer" value="Just A Guess" onClick={(e) => { this.confidenceLevelClicked(e, this.props.currentItemIndex) }}/>
-                  <input type="button" className="btn btn-check-answer" value="Pretty Sure" onClick={(e) => { this.confidenceLevelClicked(e, this.props.currentItemIndex) }}/>
-                  <input type="button" className="btn btn-check-answer" value="Very Sure" onClick={(e) => { this.confidenceLevelClicked(e, this.props.currentItemIndex) }}/>
-                </div>
-                );
-    } /*else {
-      return <div className="lower_level"><input type="button" className="btn btn-check-answer" value="Check Answer" onClick={() => { AssessmentActions.checkAnswer()}}/></div>
-    }*/
+    e.preventDefault();
+    this.props.submitAssessment();
   }
 
   getNavigationButtons() {
-    if (!this.props.shouldShowNextPrevious && this.props.confidenceLevels) {
-      return "";
-    }
-
     return <div className="confidence_wrapper">
-      {this.getPreviousButton()}
-      {this.getNextButton()}
-    </div>
+              {this.getPreviousButton()}
+              {this.getNextButton()}
+            </div>;
   }
 
   getNextButton() {
-    var disabled = (this.props.currentItemIndex == this.props.questionCount - 1) ? "disabled" : "";
+    let disabled = (this.props.currentItemIndex == this.props.questionCount - 1);
     return (
-        <button className={"btn btn-next-item " + disabled} onClick={(e) => { this.nextButtonClicked(e); }}>
+        <button
+          className="next-btn"
+          onClick={(e) => { this.nextButtonClicked(e); }}
+          disabled={disabled}
+        >
           <span>Next</span> <i className="glyphicon glyphicon-chevron-right"></i>
         </button>);
   }
 
   getPreviousButton() {
-    var prevButtonClassName = "btn btn-prev-item " + ((this.props.currentItemIndex > 0) ? "" : "disabled");
+    let disabled = (this.props.currentItemIndex === 0);
     return (
-        <button className={prevButtonClassName} onClick={(e) => { this.previousButtonClicked(e); }}>
+        <button
+          className="prev-btn"
+          onClick={(e) => { this.previousButtonClicked(e); }}
+          disabled={disabled}
+        >
           <i className="glyphicon glyphicon-chevron-left"></i><span>Previous</span>
         </button>);
   }
 
+  getSubmitButton(){
+    let submitButton = "";
```

Comment:
That is a remnant from the old implementation. I believe React handles the two the same way, and just doesn't render a component. I have preferred returning undefined because it seems cleaner to me. I just changed this to return undefined like the others.